recipe/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  $R CMD INSTALL --build .
+else
+  mkdir -p $PREFIX/lib/R/library/rerddap
+  mv * $PREFIX/lib/R/library/rerddap
+  if [[ $target_platform == osx-64 ]]; then
+    pushd $PREFIX
+      for libdir in lib/R/lib lib/R/modules lib/R/library lib/R/bin/exec sysroot/usr/lib; do
+        pushd $libdir || exit 1
+          for SHARED_LIB in $(find . -type f -iname "*.dylib" -or -iname "*.so" -or -iname "R"); do
+            echo "fixing SHARED_LIB $SHARED_LIB"
+            install_name_tool -change /Library/Frameworks/R.framework/Versions/3.5.0-MRO/Resources/lib/libR.dylib "$PREFIX"/lib/R/lib/libR.dylib $SHARED_LIB || true
```

Comment:
Is this really needed? Also, the versioned path may be a problem.

Reply:
Hi @ocefpaf, well that is not needed... all my update PRs are based on a regeneration of the recipe by https://github.com/bgruening/conda_r_skeleton_helper aka `conda-skeleton cran` and a careful git selection... this procedure  updates a bit more then required, but also ensures that not only the version + hash is updated as done by the bot. The lines you are referring to are part of the current default build.sh given by the r skeleton template. The `else` case is afaik for binary packaging which is afaik not used by conda-forge ...

Reply:
While I'm not a fan of the extra complexity I guess you need that then. Let's merge it.